packages/lib/server/repository/booking.ts

```diff
@@ -337,6 +337,23 @@ export class BookingRepository {
               },
             },
           },
+          {
+            eventType: {
+              parent: {
+                team: {
+                  members: {
+                    some: {
+                      userId,
+                      accepted: true,
+                      role: {
+                        in: ["ADMIN", "OWNER"],
+                      },
+                    },
+                  },
+                },
+              },
+            },
+          },
```

Comment:
Currently, a team admin/owner is not able to mark a no-show if the eventType is managed